src/linear_solver/helper_functions.cpp

```diff
@@ -19,9 +19,29 @@ using namespace Eigen;
 /// @param x Full Epetra vector to copy
 /// @param x_eig Empty Eigen::MatrixXd
 void epetra_to_eig_vec(int col, Epetra_Vector &x, Eigen::MatrixXd &x_eig){
+  /// Gather local information
```

Comment:
Might be better to use `//` for comments which are not meant to appear in doxygen.

Reply:
Changed, I have also scanned the other files changed in this commit to make sure that this issue isn't present in other files.